Cargo.toml

```diff
@@ -18,7 +18,7 @@ home = "0.5.3"  # use same ver in build-dep
 env_logger = "0.8"
 
 [build-dependencies]
-ureq = "2.1"
+ureq = "<2"  # allows to keep MSRV 1.41.1
```

Comment:
Shouldn't this be `1.0`? `<2` means all versions from `0.0...1` to `1.x.y.z` will work which I doubt is the case (but I don't know `ureq` so it could be true).

Reply:
Yeah, `1.0` would have been better, added an issue